dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/WorkloadDriver.java

```diff
@@ -151,9 +177,21 @@ public static void main(String[] args) throws Exception {
     return (Class<? extends WorkloadMapper>) mapperClass;
   }
 
+  private Class<? extends WorkloadReducer> getReducerClass(String className) throws ClassNotFoundException {
```

Comment:
I wonder if it's possible reduce code duplication between `getReducerClass` and `getMapperClass`?